docs/concepts/Package-Versioning.md

```diff
@@ -163,6 +163,9 @@ Always specify a version or version range for package dependencies in project fi
 | \*-\* | 1.1.0 <br> 1.1.1 <br> 1.1.2-alpha <br> 1.3.0-beta  | 1.3.0-beta | The highest version including the not stable versions. | Available in Visual Studio version 16.6, NuGet version 5.6, .NET Core SDK version 3.1.300 |
 | 1.1.\*-\* | 1.1.0 <br> 1.1.1 <br> 1.1.2-alpha <br> 1.1.2-beta <br> 1.3.0-beta  | 1.1.2-beta | The highest version respecting the pattern and including the not stable versions. | Available in Visual Studio version 16.6, NuGet version 5.6, .NET Core SDK version 3.1.300 |
 
+> [!Note]
+> Whether a package is listed does not affected the floating version resolution.
```

Comment:
Should we expand on this?

We're trying to say that floating version resolution doesn't care if it's listed or not right?

Maybe:

`Floating version resolution does not take into account whether or not a package is listed or unlisted. It will find the most suitable match for the floating version defined.`

Reply:
I like the first sentence better than what I have.

Do we need the 2nd sentence? To me it sounds a bit redundant.

Reply:
Does it mean if can resolve to unlisted version?

Reply:
Yes, it means exactly that.